app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -15,12 +15,12 @@
   heading_classes = %w(govuk-fieldset__heading)
   heading_classes << "gem-c-radio__heading-text" if heading_level == 'h1'
 
-  if ['s', 'm', 'l', 'xl'].include?(heading_size)
+  if (shared_helper.valid_heading_size?(heading_size))
     size = heading_size
-  elsif heading_level == 'h1'
-    size = 'xl'
+  elsif heading_level == "h1"
+    size = "xl"
   else
-    size = 'm'
+    size = "m"
```

Comment:
Suggestion for brevity (I haven't checked that this works though)

```
size = "m"
size = heading_size if (shared_helper.valid_heading_size?(heading_size))
size = "xl" if heading_level == "h1"
```

Reply:
I think a refactor is outside the scope of this pull request. I don't really want to rewrite things that aren't the search component, aside from adding in the helper.